pkg/users/account-logs-panel.jsx

```diff
@@ -18,58 +18,57 @@
-                            if (line.includes('tmux') || line.includes('screen')) {
-                                return;
-                            }
+    useInit(() => {
+        cockpit.spawn(["/usr/bin/last", "--time-format", "iso", "-n", 25, name], { environ: ["LC_ALL=C"] })
+                .then(data => {
+                    let logins = [];
+                    data.split('\n').forEach(line => {
+                        // Exclude still logged in and non user lines
+                        if (!line.includes(name) || line.includes('still')) {
+                            return;
+                        }
+                        // Exclude tmux/screen lines
+                        if (line.includes('tmux') || line.includes('screen')) {
+                            return;
+                        }
 
-                            // format:
-                            // admin    web console  ::ffff:172.27.0. 2021-09-24T09:02:13+00:00 - 2021-09-24T09:04:20+00:00  (00:02)
-                            const lines = line.split(/ +/);
-                            const ended = new Date(lines[lines.length - 2]);
-                            const started = new Date(lines[lines.length - 4]);
-                            const from = lines[lines.length - 5];
-                            if (isNaN(started.getTime()) || isNaN(ended.getTime())) {
-                                return;
-                            }
+                        // format:
+                        // admin    web console  ::ffff:172.27.0. 2021-09-24T09:02:13+00:00 - 2021-09-24T09:04:20+00:00  (00:02)
+                        const lines = line.split(/ +/);
+                        const ended = new Date(lines[lines.length - 2]);
+                        const started = new Date(lines[lines.length - 4]);
+                        const from = lines[lines.length - 5];
+                        if (isNaN(started.getTime()) || isNaN(ended.getTime())) {
+                            return;
+                        }
 
-                            logins.push({
-                                started,
-                                ended,
-                                from
-                            });
+                        logins.push({
+                            started,
+                            ended,
+                            from
                         });
-
-                        // Only show 15 login lines
-                        logins = logins.slice(0, 15);
-                        setLogins(logins);
-                    })
-                    .catch((ex) => {
-                        console.error(ex);
                     });
-        }
-    }, [logins, name]);
+
+                    // Only show 15 login lines
+                    logins = logins.slice(0, 15);
+                    setLogins(logins);
+                })
+                .catch((ex) => {
+                    console.error(ex);
+                });
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18198-20230118-092548-536125bd-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)